app/src/main/java/com/github/fribourgsdp/radio/external/musixmatch/MusixmatchLyricsGetter.kt

```diff
@@ -176,6 +177,15 @@ object MusixmatchLyricsGetter : LyricsGetter {
                 .replace("\n", "<br>")
     }
 
+    /**
+     * Cancels the effect of previous function, replacing marked name of song with [REPLACEMENT_STRING]
+     */
+    fun makeReadable(lyrics: String) : String{
+        return lyrics
+            .replace("<br>", "\n")
+            .replace(Regex("<strike>.+</strike>"), REPLACEMENT_STRING)
+    }
+
```

Comment:
it seems a little wonky to strike then delete the striked text instead of directly replacing the text but I understand your reasoning. Regardless I would change the documentation to something more explicit such as "replaces striked words in lyrics with a replacement string" because the current documentation is not very explicit and "previous function" does not make much sense when someone does not want to specifically look at the code, which is exactly what the documentation is for.

Reply:
I agree, if we reorder the members of this class, this doc does not make sense anymore.